INSTALL.md

```diff
@@ -99,13 +114,43 @@ The initialization command will prompt for the following information.
    prompts for its password.  This information will be needed to later connect
    to Accumulo.
 
-Multiple Nodes
---------------
+## Run Accumulo
+
+There are two primary methods for running Accumulo:
+
+1. Run individual Accumulo services using `accumulo-service`. Useful if you are
```

Comment:
I'm not sure this is a good recommendation. The accumulo-service scripts do additional things, like logging, and PID file management, which are not suitable for all init systems. Instead of mentioning this as a "primary method" for running Accumulo, we should simply explain the extra functionality it provides, and leave it to the user to decide when they want to make use of that functionality (with suggestions that it may be useful with Ansible, etc.).